pkg/controller/beat/name.go

```diff
@@ -0,0 +1,26 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package beat
+
+import (
+	commonbeat "github.com/elastic/cloud-on-k8s/pkg/controller/common/beat"
+	common_name "github.com/elastic/cloud-on-k8s/pkg/controller/common/name"
+)
+
+// namer is a Namer that is configured with the defaults for resources related to a Beat resource.
+var namer = common_name.NewNamer("beat")
+
+type Namer struct {
+}
```

Comment:
This is to allow it to implement `pkg/controller/common/beat/name.go`. And that interface is there so that if a different controller wants to deploy Beats, they can use a different naming scheme. E.g. suffix names with `stackmonitoring` instead of `beat`.